### PR TITLE
fix: address loading issue

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
In this PR, I have made some minor fixes:
- fixed address management bug
- fix the form validation issue
- fix private routing
### [Live link](https://dazzle-cerashop-git-temp-dev-apurvasawant11.vercel.app/)

### CODE REVIEW

**Review by mikesch803 — APPROVED** [review]
looks good to me, great job that you implemented all the features, nothing to suggest

File: src/pages/cart/CartSummary.jsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ const CartSummary = () => {
   const { cartList, couponDetails, setCouponDetails } = useCart();
   const { orderDispatch } = useOrder();
   const { address } = useData();
+  console.log(address);
```
Comment (hsnice16) [inline src/pages/cart/CartSummary.jsx]: If this `console` is not needed, you can remove this.

**Review by hsnice16 — APPROVED** [review]
Great work. Just suggest one change. Good Job.

File: src/pages/cart/CartSummary.jsx (hunk 1 above)
Comment (ApurvaSawant11) [reply src/pages/cart/CartSummary.jsx]: Yes...  I will remove this. Thanks

**Pull request merged**